agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -969,6 +969,16 @@ bool backhaul_manager::backhaul_fsm_main(bool &skip_select)
         eth_link_poll_timer = std::chrono::steady_clock::now();
         m_eth_link_up       = network_utils::linux_iface_is_up_and_running(m_sConfig.wire_iface);
         FSM_MOVE_STATE(OPERATIONAL);
+
+        if (backhaul_sta_steering_enable) {
```

Comment:
This needs a comment IMO. Something like:

```
// This event may come as a result of enabling the backhaul, but also as a result
// of steering. *Only* in case it was the result of steering, we need to send a steering
// response.
```

Reply:
Accepted